exonum-java-binding/bom/pom.xml

```diff
@@ -28,7 +28,13 @@
         <artifactId>exonum-java-binding-common</artifactId>
         <version>${project.version}</version>
       </dependency>
-      
+
+      <dependency>
+        <groupId>com.exonum.binding</groupId>
+        <artifactId>exonum-java-binding-time-service</artifactId>
```

Comment:
exonum-java-binding-time-service -> exonum-time-service/exonum-time-oracle/exonum-time?

The original crate name is "exonum-time", but in the docs it is mainly mentioned as "Time oracle": https://exonum.com/doc/version/latest/advanced/time/ 

@exonum/enterprise , @slowli which name would you consider the best? The shortest "exonum-time"? Or "exonum-time-oracle"? Or "-service"?

Reply:
IMO exonum-time-service